src/analysis/retail/paladin/holy/modules/features/Abilities.tsx

```diff
@@ -337,14 +350,30 @@ class Abilities extends CoreAbilities {
         isDefensive: true,
       },
       {
-        spell: SPELLS.BLESSING_OF_PROTECTION.id,
+        spell: TALENTS.BLESSING_OF_PROTECTION_TALENT.id,
         category: SPELL_CATEGORY.UTILITY,
-        cooldown: 5 * 60,
+        cooldown:
+          // Reduces the cooldown of Blessing of Protection by 60
+          60 * (combatant.hasTalent(TALENTS.UNBREAKABLE_SPIRIT_TALENT.id) ? 4 : 5),
```

Comment:
I would recommend having a constant like:
```tsx
const UNBREAKABLE_SPIRIT_SCALING = [0, 60];
```
and then referencing it with something like:
```tsx
cooldown: 300 - UNBREAKABLE_SPIRIT_SCALING[combatant.getTalentRank(TALENTS.UNBREAKABLE_SPIRIT_TALENT)]
```

This one is more personal preference on representation though.

Reply:
would second this ^

Reply:
Done